File: dev-page/src/main/java/org/example/page/task/expert/user/signup/AssemblerExpertUserSignUp.java
package org.example.page.task.expert.user.signup;

import org.example.core.page.presentation.AbstractIPage;

public class AssemblerExpertUserSignUp {
    public static AbstractIPage getPage() {
        final var request = new Request();
        final var service = new Service(request);
        final var controller = new Controller(service);
        return new Page(controller);
    }
}

File: dev-page/src/main/kotlin/org/example/page/task/expert/user/signup/AssemblerExpertUserSignUp.kt
package org.example.page.task.expert.user.signup

import org.example.core.page.presentation.AbstractIPage

object AssemblerExpertUserSignUp {
    val page: AbstractIPage
        get() {
            val request: Request = Request()
            val service: Service = Service(request)
            val controller: Controller = Controller(service)
            return Page(controller)
        }
}
